food-bank-app/src/components/PriorityQueueItem/PriorityQueueItem.tsx

```diff
@@ -0,0 +1,157 @@
+import { Icon } from '@rneui/base'
+import { useTheme } from '@rneui/themed'
+import { useState } from 'react'
+import { Image, Text, View, TouchableOpacity, StyleSheet } from 'react-native'
+import Modal from 'react-native-modal'
+const nopictureUrl = 'https://upload.wikimedia.org/wikipedia/en/6/60/No_Picture.jpg'
```

Comment:
```suggestion
const NO_PICTURE_URL= 'https://upload.wikimedia.org/wikipedia/en/6/60/No_Picture.jpg'
```
This would require changing all coincidences, though